clusters/build-clusters/01_cluster/assets/prom_prow_rules.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  annotations:
+    source: https://github.com/loodse/prow-dashboards/blob/e4af4c0ec879a0971591adcfc1e6f52c2ff015ee/prometheus-rules.yaml
+  name: prow-rules
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: prow
+    rules:
+    # These metrics are based on generic pods, not just Prow jobs.
+
+    # group interesting information into a single metric
+    # squash metrics from multiple kube-state-metrics pods
+    # {namespace="...",pod="...",node="...",pod_ip="1.2.3.4",phase="..."} 1
+    - record: prow:pod
+      expr: |
+        max by (namespace, pod, node, pod_ip) (kube_pod_info) *
```

Comment:
Don't we a created-by-prow label selector here

Reply:
that requires another join with `kube_pod_labels`, do you know how to do that?

Reply:
You should just be able to do 

```
(kube_pod_info{label_created_by_prow="true"})
```

Reply:
No, the label is not on the `kube_pod_info` metric, its only on `kube_pod_labels` and would require joining on namespace and name (And I don't know how joining in promQL works, the rules here are all copied)

Reply:
Ah, ew